### PR TITLE
doc: add travel fund allocation for 2021

### PR DESCRIPTION
**Pull request by mhdawson**
Add amount approved by board for 2021

### CODE REVIEW

File: project-resources/MEMBER_TRAVEL_FUND.md [hunk 1]
```diff
@@ -209,4 +209,12 @@ The request from the
 [Cross Project council](https://github.com/openjs-foundation/cross-project-council)
 for a 2020 joint travel fund was approved in the amount of $60,000.
 
+
+## 2021 OpenJS Foundation Board of Directors Allocation
+
+The request from the
+[Cross Project council](https://github.com/openjs-foundation/cross-project-council)
+for a 2020 joint travel fund was approved in the amount of $20,000 (lower this year
```
Comment (mhdawson) [inline project-resources/MEMBER_TRAVEL_FUND.md:217]:
```
for a 2021 joint travel fund was approved in the amount of $20,000 (lower this year
```

**Review by joesepi — APPROVED** [review]
Thanks for the update.

I will also have some travel fund housekeeping updates that I started today. Simply moving 2020 to archived and adding a new 2021 file and updating links and adding a very simple readme to the travel fun directory pointing to the documentation in the project-resources file.

**Comment by brianwarner** [comment]
This time around I would also recommend using the [LFX crowdfunding tool](https://crowdfunding.lfx.linuxfoundation.org/#projects) to handle actual reimbursement. It can be set up specifically for travel funds, and can also receive donations from organizations other than the Foundation.

To my knowledge it can't replace the actual decision process (e.g., CPC still needs a way to track and approve applications, and avoid overspending the budget) but it does provided a streamlined process to actually get paid, and provides a transparent ledger.

**Review by christian-bromann — APPROVED** [review]

**Review by tobie — CHANGES_REQUESTED** [review]
The Member Travel Fund allocation process focusses on helping underserved communities in particular. If there is a year in which underserved communities are suffering [it is this year](https://www.cdc.gov/coronavirus/2019-ncov/community/health-equity/race-ethnicity.html).

In this context, cutting down this fund without reassigning the savings into a program that does the same kind of work sends the wrong signal and isn't aligned with the values we are trying to defend, notably in #699.

While we don't necessarily have a program setup just yet, we should request from the Board a formal commitment to earmark the saved funds for DEI-related work and have it mentioned in this PR.

**Comment by mhdawson** [comment]
@tobie this was discussed in latest Board finance committee meeting, and the a recommendation was made to the board. Based on that recommendation the board approved changing the total back to $60k, changing the budget line item to `community fund` with the expectation the the CPC will propose additional review/approval processes for spending the money beyond just travel.  

I think next steps after this lands is for CPC to work on proposals for other ways the fund may be used.

**Review by tobie — APPROVED** [review]
This is a great outcome! Thank you for your advocacy and the Board members for their understanding.

**Comment by tobie** [comment]
> I think next steps after this lands is for CPC to work on proposals for other ways the fund may be used.

Tracking this in #756.

**Comment by mhdawson** [comment]
I'll leave this until next CPC meeting so we can give an update to the full team before landing.

**Comment by tobie** [comment]
Update was given to the CPC. Tracking the proposal that the CPC has to make in #756. This can be merged.

**Pull request merged**